_test/test_mpi_wrappers/check_progress_common_methods.m

```diff
@@ -0,0 +1,330 @@
+classdef check_progress_common_methods< TestCase
```

Comment:
`test_*` will be picked by the test suite and will be attempted to run. This will fail, as `check_progress_common_methods` need additional input, specifying what particular class common methods are tested. The subclasses define the particular class and are all started from `test_` to run `check_progress_common_methods` methods with actual test class helpers

`test_` is inappropriate and will fail here